appengine/redis/server.js

```diff
@@ -25,42 +27,39 @@ nconf.argv().env().file('keys.json');
 // [START client]
 // Connect to a redis server provisioned over at
 // Redis Labs. See the README for more info.
-var client = redis.createClient(
+const client = redis.createClient(
   nconf.get('redisPort') || '6379',
   nconf.get('redisHost') || '127.0.0.1',
   {
     'auth_pass': nconf.get('redisKey'),
     'return_buffers': true
   }
-).on('error', function (err) {
-  console.error('ERR:REDIS: ' + err);
-});
+).on('error', (err) => console.error('ERR:REDIS: ' + err));
```

Comment:
No, in fact I'm making `err` a second argument. I have no idea what `err` will be, and I don't want to risk it getting stringified as `[Object object]`